### PR TITLE
[ADD] New module 'website_google_tag_manager'

### PR DESCRIPTION
**Pull request by sebalix**
Google Tag Manager
==================

This module allows to configure your Odoo website to support the *Google Tag Manager* tool.

Configuration
-------------

To configure this module, you need to:

* go to `Website Admin/Configuration`
* fill in your `Google Tag Manager Key` (e.g. `GTM-ABCDEF`)

### CODE REVIEW

**Comment by lasley** [comment]
@sebalix - a rebase on current base should fix your Travis

**Comment by sebalix** [comment]
@lasley thanks, Travis and runbot are now green

File: website_google_tag_manager/views/templates.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 ABF OSIELL <http://osiell.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<openerp>
```
Comment (lasley) [inline website_google_tag_manager/views/templates.xml]: `<odoo>` and can remove data tags (in all files)

**Review by lasley — APPROVED** [review]
Thanks @sebalix - minor comment but otherwise 👍

**Comment by rafaelbn** [comment]
Hi @sebalix thanks for this module. When you make a PR please paste in the description of the PR de description of the module, this helps a lot to get the interest in your PR because when you receive the email you can read what if the module for 😉

**Review by rafaelbn — APPROVED** [review]

File: website_google_tag_manager/views/templates.xml (hunk 1 above)
Comment (sebalix) [reply website_google_tag_manager/views/templates.xml]: @lasley done

**Comment by sebalix** [comment]
@rafaelbn I'm used to do so, I just forgot it ;) Updated.

**Review by susport — COMMENTED** [review]
👍

**Comment by lasley** [comment]
Thanks @sebalix

**Pull request merged**